packages/luna/src/locales/ar-eg.json

```
@@ -758,5 +758,118 @@
   "error.bio.empty": "صف نفسك في كلمات موجزه.",
   "error.bio.short": "محتوي البايو قصير للغاية.",
   "error.bio.long": "تخطي البايو الحد الأقصي من الكلمات.",
-  "error.bio.invalid": "يحتوي البايو علي محتوي غير مناسب."
+  "error.bio.invalid": "يحتوي البايو علي محتوي غير مناسب.",
+  "error.api.forbidden": "لا يمكنك الوصول لهذه المعلومات، برجاء المحاولة مرة أخري",
```

```suggestion
  "error.api.forbidden": "لا يوجد صلاحيات كافيه للوصول لتلك المعلومات، الرجاء استخدام حساب اخر او التواصل مع أحد المسؤلين",
```

خلي بالك هنا من صيغة الكلام٫ انا قلت المشكله في الصلاحيات مش في المستخدم. حاجه بسطه لكن بتفرق

---

Let's put a dot at the end of every error message.